src/controllers/confirmation.controller.ts

```diff
@@ -0,0 +1,65 @@
+import { NextFunction, Request, Response } from "express";
+import {formatDateForDisplay} from "../client/date.formatter";
+import {PTFCompanyProfile} from "../model/company.profile";
+import * as templatePaths from "../model/template.paths";
+import * as sessionService from "../services/session.service";
+import {ALREADY_SUBMITTED, COMPANY_NUMBER, COMPANY_PROFILE, SIGN_IN_INFO, USER_PROFILE} from "../session/keys";
```

Comment:
ALREADY_SUBMITTED, COMPANY_NUMBER and SIGN_IN_INFO . are unused

Reply:
Gone